markdown/articles/2020/03/22/DDFAQs.md

```diff
@@ -0,0 +1,600 @@
+combo itself and so ensuring these are at a maximum is key. You don't want to be having to
+"burn" through cantrips to find pieces to then find yourself unable to draw into an otherwise
+game-winning pile.
+
+As a result I would expect any Doomsday deck to be running {{Brainstorm}}, {{Ponder}} and {{Preordain}}
+in various ratios of 4:4:X. In addition to this some lists will be running the previously cited {{Arcum's Astrolabe}}
+along with some spicier elements such as {{Street Wraith}} and {{Edge of Autumn}} however more on those later.
+Combine these with some of the cantripping protection elements and more niche "combo cards" like
+{{Conjurer's Bauble}} you should end up with a very high cantrip number.
+
+**DDFT
+
+I would expect a modern DDFT list to run the default 8 of Ponder x Brainstorm and anywhere between
+2 and 4 copies of Preordain. I would also not be surprised to expect maybe 0-2 Street Wraith. If
+Astrolabe is to be run, it would likely take the Preordain slot as the deck needs room for its fast mana.
+
+**DDEFT
+
+DDEFT would be also wanting to run the 8 plus 2-4 Preordain. Unlike DDFT however it cannot afford to run
+either cycling effects like Street Wraith, nor can it run Arcum's Astrolabe. With {{Experimental Frenzy}}
+out you cannot easily cast either card from the top of your deck thus increasing the liklihood of
+"whiffing" whilst with a Frenzy out. This shouldn't matter however as a Frenzy can itself generate
+such a high potential card advantage that having less cantrips is considered fine.
+
+**Meandeck
+
+Meandeck probably wants the highest number of cantrips of any of the variants. It looks to play a longer
+game which means trying to hit land drops most consistently along with finding the normal combo elements.
+Meandeck should run the standard 8 plus 3-4 Preordain. Unlike DDFT any Astrolabe being run would be in 
+addition to Preordains, not instead of. The lists also run between 1-2 Street Wraith and 1-2 Edge of
+Autumn.
+
+
+### 2.4 Protection
+
+No combo deck wants to run into potential permission without first having some sort of guard up (except 
+maybe Belcher). Interaction is part of what makes Legacy so much fun and having tools to interact with
+your opponent proactively, or to react to their interaction are crucial to have. This section touches
+somewhat upon the later discussion of splash colours but hopefully will keep it to a simple numbers
+game for now.
+
+Interaction can come in many forms such as proactive discard effects like {{Duress}} and {{Thoguhtseize}},
+reactive cards like {{Force of Will}} and {{Flusterstorm}}, cards that can perform both roles to an 
+extent like {{Orim's Chant}} and {{Veil of Summer}} or static abilities such as {{Defense Grid}} and 
+{{Teferi, Time Raveler}}. Depending on the list you are running these are all options available to a 
+Doomsday pilot. Normally you would want at least 8 pieces of permission maindeck with additional
```

Comment:
"permission" maybe should be "permission/protection" here

Reply:
As above.